I.write-tests/01-greet-people/greet-people.test.js

```diff
@@ -1,7 +1,17 @@
+var greetPeople = require("../01-greet-people/greet-people")
```

Comment:
I think this path is a bit over-specified. You should be able to change it to `'./greet-people'`. A single dot means "look in this folder". 2 dots mean "look in the parent folder".

So in this case, you're looking in the parent folder, then back to current folder and then looking at the file. This can be simplified to just look in the current folder for the file.